ColorMemoryBook/Sources/Views/HomeViewController.swift

```diff
@@ -120,28 +147,54 @@ class HomeViewController: BaseViewController {
         tagSearchTextField.leftViewMode = .always
         tagSearchTextField.rightView = clearButton
         tagSearchTextField.rightViewMode = .whileEditing
-        //한번 포커싱되면 텍스트 없어도 x버튼 나타남, 다른 걸로 바꿔야
     }
     
-}
-
- 
-extension HomeViewController: UICollectionViewDelegateFlowLayout, UICollectionViewDataSource{
-    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return 300
+    @objc private func floatingButtonDidTap(_ sender: UIButton){
+        floatingButton.isSelected.toggle()
+        if floatingButton.isSelected{
+            floatingButton.setImage(UIImage(named: "floating_black"), for: .normal)
+            colorFilterButton.isHidden = false
+            memoryRecordButton.isHidden = false
+        }
+        else{
+            floatingButton.setImage(UIImage(named: "floating_blue"), for: .normal)
+            colorFilterButton.isHidden = true
+            memoryRecordButton.isHidden = true
+        }
```

Comment:
```suggestion
        floatingButton.isSelected.toggle()
        colorFilterButton.isHidden = sender.isSelected
        memoryRecordButton.isHidden = sender.isSelected
```
이렇게 하고
floatingButton의 setImage에서
```swift
floatingButton.setImage(UIImage(named: "floating_blue"), for: .normal)
floatingButton.setImage(UIImage(named: "floating_black"), for: .selected)
```
이렇게 했을듯!

Reply:
👍👍